src/main/java/com/chetanmeh/oak/index/config/generator/IndexConfigGenerator.java

```diff
@@ -62,8 +65,28 @@ public NodeTypeInfoProvider getNodeTypeInfoProvider() {
         };
     }
 
+    public static boolean isXPath(String query) {
```

Comment:
Instead of copy & paste code, can we reference the existing method?

Reply:
Yes this was from before I could update the oak version. Had some issues with that before.